index.bs

```diff
@@ -692,6 +678,23 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
 
 </div>
 
+<div algorithm="pullBidirectionalStream">
+To <dfn>pullBidirectionalStream</dfn>, given a {{WebTransport}} object <var>transport</var>, run
+these steps.
+
+1. Let |session| be |transport|'s [=[[Session]]=].
+1. Let |p| be a new promise.
+1. Return |p| and run the following [=in parallel=].
+1. Wait until there is an [=session/receive a bidirectional stream|available incoming bidirectional
+   stream=].
+1. Let |internalStream| be the result of [=session/receiving a bidirectional stream=].
+1. Let |stream| be the result of [=BidirectionalStream/creating=] {{BidirectionalStream}} with
```

Comment:
We need to queue a task for this and the remaining steps, to get back on the JS thread to create these WebIDL objects.

Reply:
Done.